scripts/autogenerate_files.py

```diff
@@ -88,24 +88,77 @@ def gen_c_zetas():
         zeta.append(signed_reduce(pow(root_of_unity, i, modulus) * montgomery_factor))
 
     # The source code stores the zeta table in bit reversed form
-    yield from (zeta[bitreverse(i, 7)] for i in range(128))
+    zeta_reversed = []
+    for i in range(128):
+        zeta_reversed.append(zeta[bitreverse(i, 7)])
+    # return a finite sequence here, not a generator
```

Comment:
Why was this changed?

Reply:
Nit: You can keep it was it was, but use `zetas = list(gen_c_zetas())` on the caller.

Reply:
Oh... I thought it had to be a sequence not a genertor, so it could be indexed. Happy to defer to whatever is considered better style in Python.

Reply:
> I thought it had to be a sequence not a genertor

That's true, but you can convert the generator into a list with `list(...)` at the caller.

Reply:
Would you strongly prefer to change this back, or leave it as is?